demo/xgboost-demo/README.md

```diff
@@ -0,0 +1,27 @@
+This folder contains files demonstrating how to work with xgboost.
+
+## xgboost.txt
+The sample training data utilized in the main demo folder but in LibSVM format which is used by xgboost.
```

Comment:
Are there other demo files? This seems to refer to other files up a directory. If we're only going to include an xgboost demo we should eliminate the extra folder structure.

Reply:
@JohannesDaniel  thoughts on this?    Let's make a decision and get this merged so we can merge the docs!

Reply:
This is the source of the files: https://github.com/o19s/elasticsearch-learning-to-rank/tree/main/demo

Reply:
xgboost.txt is included: https://github.com/JohannesDaniel/opensearch-learning-to-rank-base/blob/add-ltr-demo/demo/xgboost-demo/xgboost.txt

Reply:
I just copied exactly what I found in the link posted above. But I agree with @sstults something might be missing. In the official documentation we have the following link: https://github.com/JohannesDaniel/opensearch-learning-to-rank-base/blob/add-ltr-demo/demo/xgboost-demo/xgboost.txt

But this file is not included here.

Reply:
I think we're fine. It's not that big of a deal.